lib/txmgr/txmgr.go

```diff
@@ -21,9 +21,19 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 )
 
+// TxError is a typed error for transaction errors.
+type TxError string
+
+func (e TxError) Error() string {
+	return string(e)
+}
+
 const (
 	// PriceBump geth requires a minimum fee bump of 10% for regular tx resubmission.
 	PriceBump int64 = 10
+
+	// ErrIntrinsicGasTooLow is the error message returned by json-rpc when the intrinsic gas is too low.
+	ErrIntrinsicGasTooLow TxError = "intrinsic gas too low"
```

Comment:
no need for this error, please use geth [error](https://github.com/ethereum/go-ethereum/blob/master/core/error.go#L81) instead